kbi/0025/index.rst

```diff
@@ -0,0 +1,58 @@
+.. index::
+   single: shared repository; multi-user
+
+KBI0025:  Saving changes in datasets owned by other users
+=========================================================
+
+:authors: Adina Wagner <adina.wagner@t-online.de>
+:discussion: <link>
+:keywords: sharedRepository, git config, multi-user
+:software-versions: datalad_0.19.0, git_2.39.2
+
+It is generally recommended to not share one and the same dataset or Git
+repository when collaborating with others.
```

Comment:
Because "share" is such an over-loaded word, it would be good to point out that we are talking about "accessing the same directory and its content by multiple users". Another possible phrase would be "concurrently accessing the same directory and files on disk".

Reply:
```suggestion
It is generally recommended to not share one and the same dataset or Git
repository (i.e., the same directory on the file system) when collaborating with others.
```

maybe like this?